webpack/assets/javascripts/compute_resource/ovirt.js

```diff
@@ -196,3 +196,28 @@ export function datacenterSelected(item) {
   // eslint-disable-next-line no-undef
   testConnection($('#test_connection_button'));
 }
+
+export function vnicSelected(item) {
+  const selectedVnicProfile = $(item).val();
+  const vnicOptions = JSON.parse(
+    $('select[id$=_vnic_profile]')[1].getAttribute('data-profiles')
```

Comment:
you can add `vnic` to https://github.com/theforeman/foreman/blob/develop/.eslintrc#L13 so lint doesn't complain